uk.co.compucorp.civicrm.hrsampledata/CRM/HRSampleData/Cleaner/OptionValue.php

```diff
@@ -13,10 +13,31 @@ class CRM_HRSampleData_Cleaner_OptionValue extends CRM_HRSampleData_CSVCleanerVi
    *   Should at least contain `option_group_id` & `name`
    */
   public function visit(array $row) {
+    if ($row['option_group_id_type'] == 'title') {
+      $row['option_group_id'] = $this->getOptionGroupNameByTitle($row['option_group_id']);
+    }
+    unset($row['option_group_id_type']);
+
     $this->deleteRecord(
       'OptionValue',
       ['name' => $row['name'], 'option_group_id' => $row['option_group_id']]
     );
   }
 
+  /**
+   * Gets option group name by its title
+   *
+   * @param $groupTitle
+   */
+  private function getOptionGroupNameByTitle($groupTitle) {
+    $optionGroup = $this->callAPI( 'OptionGroup', 'get', [
+      'sequential' => 1,
+      'return' => ["name"],
```

Comment:
Please use single quotes for name, just to keep it consistent with the other strings